src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: (2 * $calendar-horizontal-padding) $calendar-horizontal-padding;
+  box-sizing: border-box;
+  min-width: $calendar-width;
+  width: $calendar-width;
+  gap: $days-gap;
+
+  &__day {
+    position: relative;
+
+    width: $day-size;
+    height: $day-size;
+    border: 1px solid black;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: #eee;
+    transition: 0.5s;
+
+    @for $day from 1 through 31 {
+      &--#{$day} {
```

Comment:
Instead of creating separate classes for each day you could style it using **:nth-child** selector.